README.md

```diff
@@ -18,7 +18,8 @@ See the _Limitations_ section below for some of known limitations.
 
 ### Quickstart
 
-1. Checkout this repo
+0. Checkout this repo
+1. Configure Maven Settings file according to the [Jenkins Plugin Tutorial](https://wiki.jenkins.io/display/JENKINS/Plugin+tutorial) so that plugins can be properly downloaded for the pom.xml input
```

Comment:
Is this step needed always since this PR?

Reply:
It was needed even before if you wanted to get a right output.
We were just lucky since our machines are configured for Jenkins development. 

I can bundle a local settings.xml into the demo instead

Reply:
no no. Just wanted to clarify, thanks